calico-vpp-agent/cni/network_vpp.go

```diff
@@ -234,12 +234,25 @@ func (s *Server) AddVppInterface(podSpec *storage.LocalPodSpec, doHostSideConf b
 	}
 	s.log.Infof("created tun[%d]", swIfIndex)
 
-	if tun.NumRxQueues > 1 && s.NumVPPWorkers > 0 {
+	var nbDataThread int = 0
+	if config.IpsecNbAsyncCryptoThread > 0 {
```

Comment:
Maybe we should put the `if nbDataThread <= 0` condition only inside the async crypto `if`. afaik `nbDataThread==0` is legal when `workers==0`
````go
nbDataThread := (int) s.NumVPPWorkers
if config.IpsecNbAsyncCryptoThread > 0 {
       nbDataThread = nbDataThread - config.IpsecNbAsyncCryptoThread
       s.log.Infof("ipsec async crypto %d", config.IpsecNbAsyncCryptoThread)
       if nbDataThread <= 0 {
     		s.log.Error("not enough thread for async crypto")
   		nbDataThread = (int) s.NumVPPWorkers
	}
}
````